### ISSUE TITLE
Changes to frontend

### ISSUE THREAD

**Issue by LouiseABowler** [alan-turing-institute/vehicle-grid-integration-webapp-private#37]
- [x] Info. on selection ov LV, mention that shape of networks can be seen in results?
- [ ] resources section: webinar,APEN
- [ ] future work: additional networks; cost module
- [ ] Copyright to something open source

python
- [x] remove. "Primary feeder powers. This reports the same as Primary feeders’ loadings but is messier, so maybe it might be worthwhile taking out?"
- [x] remove "MV network powers"
- [ ] Remove icoo from the Profile plots
- [ ] watermarking images. see [link](https://www.tutorialspoint.com/how-to-plot-a-watermark-image-in-matplotlib)
- [ ] LV profiles and MV profiles figures. Either different sections or subfigures if not too crowded (also depends where the legend is).

- [x] highlighted they want information on the parameters and I mentioned that we're currently working on it. so hopefully what @LouiseABowler just did covers this. 
- [x] Walkthrough info attached to title: Text and image sent by email
- [x] DG: Choose or upload data for two distributed generation sites. For example a commercial solar  installation or a stand-alone solar plant. The walkthrough guide shows the network connection of the DG sites. 
- [x] FC: Choose or upload data for two urban or motorway fast charging filling stations. These stations are similar to a site from a TESLA supercharger network; or Fastned network which is what we use for the available fast charging profile in this work. The walkthrough guide shows the network connection of the FC stations sites. 
- [x] "LV connected" header. Leave empty for no green technologies, use available profiles, or upload your own data for electric vehicles, solar PV and heat pumps. Change the penetration level from 0% to 100% penetration. The higher the penetration of green technologies, the higher the impact that will be seen on the network (e.g. voltage and powers will exceed statutory limits, which is represented in red dashed lines in the relevant output figures.).
- [x] some more info on the outputs.
- [x] Acknowledgment- can they be smaller font, under the submit card? as the submit card should be the highlight and not sharing same attention as tool? 
- [ ] Website Title Vue App--> Network analysis tool
- [x] "the top bar should contain a menu, otherwise it's useless and better be removed"
- [ ] "footer - Copyright © 2019-2022 e4futuregrid.com. All rights reserved. | main contact " html code uploaded on slack
- [ ] Oscar's note on loading icon,etc [issue#57](https://github.com/alan-turing-institute/vehicle-grid-integration-webapp-private/issues/57#issue-1103245882)
- [x] include REG website on the Resources Page or embed in acknowledgment text where REG is mentioned.

Tooltips
- [x] adding information on the various elements e.g. that shows when hovering over a small information icon next to each parameter?
  - [x] Electricity distribution network parameters (Header): More information on the network models and parameters can be found in the project repositories. 
  - [x] MV transformer scaling: number by which the nominal power rating of the primary (HV/MV) power transformer are multiplied by, to allow an increase in demand before the substation is overloaded. (Affects the primary substation utilization on the ‘Transformer Powers’ figure that is returned.)
  - [x] Setpoint: nominal voltage, in ‘per-unit’, at which the voltage is held on the low-voltage side of the primary substation. The on-load tap changer will aim to keep the voltage close to this value.
  - [x] Bandwidth: in ‘per-unit’, the on-load tap changer at the primary substation will only change when the voltage passes outside of the setpoint plus-or-minus the bandwidth. A wider bandwidth means that the voltage at the primary substation will vary more before the tap changes.
  - [x] Proportion residential loads: For example 0.8 indicates that 80% of the loads on the network are residential loads and 20% are industrial & commercial loads.
  - [x] Demand and generation profiles (Header): Choose from available profiles or upload your own data. More information on the available profiles can be found in the project repositories. For "Penetration" values 1 means 100% penetration.
- [x] Prompt to give information on how the csv file should look like. e.g. "Please upload a csv file with a header+ 48 rows corresponding to 1 day at 30-minute resolution data. There is no limit on the number of columns that can be uploaded (i.e. several days of data). For an example csv file, see here." I will paste below the example csv file that we can link to. (one of the solar data).
- [x] prompt for penetration to indicate between 0 and 1.
- [ ] prompt to indicate a user can choose a minimum of 2 networks and a max of 5 networks. Or just change the error message to say "Min 2 and Max 5 networks allowed" or something to the sort

- [x] from Matt's email: the axis label ‘Power (kW, or normalised)’ should be updated to just be ‘Power, kW’

Later

- [x] Slightly increase the size of the Supergen logo to ensure all logos are same size. I just noticed it also needs increasing on my slides.
- [x] add Lloyd's logo (LRF confirmed we can use it)
- [x] Include a picture- waiting on a colleague/graphic designer to see if he can provide one on time!
- [x] smaller front so that all the sentence fits on one line "Build and simulate an electricity distribution network with EVs and other green technologies"

- [x] Csv validation
- [x] Remove penetration for smart meters
- [x] Start with LV network comparison plot open (with voltages + powers), reverse order of MV

Links:
- [x] Remove OpenDSS link
- [x] Data - link to Pam's work
- [x] Matt's original code

- [x] Transformer scaling limits. 0.5 to 4
- [x] "11kV connected electric vehicle charging profile" --> 11kV connected Fast Chargers Station
- [x] make the LV options plural i.e. smart meter--> smart meters. Heat Pump --> Heat pumps
- [x] Photovoltaic --> solar PV  
- [x] landing page- one card, bigger size, acknowledgement and logos. (in progress)
- [x] change of titles. --I included figures below with suggestions
- [x] Since we are only showing one figure at a time, is it possible to increase the size of the figure being shown? both width and Height. This will probably fix the primary powers and transformer powers graphs.
- [x] Fix when no CSV upload. Same as CSV validation?
- [x]  Various links to documentations. e.g. links to repo section containing the open data; links to Matt's networks;
- [x] Primary loadings --> Primary feeders' loadings
- [x] Primary powers --> Primary feeders' powers
- [x] Change the name of the section from Profiles to Average of Profiles
(Heat pump name changes have been moved to a separate issue as they need changing in the API code)
